test/voting_to_change_keys_test.js

```diff
@@ -290,6 +290,29 @@ contract('Voting to change keys [all features]', function (accounts) {
         {from: votingKey}
       ).should.be.rejectedWith(ERROR_MSG)
     })
+    it('cannot create a ballot with earlier removed mining key', async () => {
+      await proxyStorageMock.setVotingContractMock(accounts[0]);
+      VOTING_START_DATE = moment.utc().add(20, 'seconds').unix();
+      VOTING_END_DATE = moment.utc().add(10, 'days').unix();
+
+      await addMiningKey(accounts[1]);
+      await addVotingKey(votingKey, accounts[1]);
+
+      await addMiningKey(accounts[2]);
+      const {logs} = await keysManager.removeMiningKey(accounts[2]);
+      logs[0].event.should.be.equal("MiningKeyChanged");
+      
+      await voting.createBallot(
+        VOTING_START_DATE,
+        VOTING_END_DATE,
+        1,
+        1,
+        "memo",
+        accounts[2], // _affectedKey
+        accounts[1],
+        {from: votingKey}
+      ).should.be.rejectedWith(ERROR_MSG);
+    })
```

Comment:
Let's add here one more call of `createBallot` for adding some another miningKey. This call must be successful.